src/download/src/lib.rs

```diff
@@ -175,6 +181,10 @@ pub mod curl {
                 .connect_timeout(Duration::new(30, 0))
                 .chain_err(|| "failed to set connect timeout")?;
 
+            handle
+                .ssl_verify_peer(!use_unsafe_ssl())
+                .chain_err(|| "failed to configure unsafe SSL mode")?;
```

Comment:
Will this error message be misleading if we're trying to configure *safe* SSL mode and something goes wrong?

Reply:
It will, I'll wrap this in an `if`.

Reply:
Sounds good, thanks.